### DISCUSSION TITLE
Manual trigger flow form

### DISCUSSION THREAD

**Discussion by JonL1** [directus/directus#14817] [Archive]
There is an option to trigger a manual flow via a button on table and record view but it's static in nature. You click and flow runs as designed in the flow editor. 

I'm proposing to be able to trigger manually a flow and pass parameters based on a form.

You click the flow button a form opens and lets you add values as input parameters that will be used in the flow logic. You can customise the form with different UI input element and values.

Additionally this would improve flow reusability.

Just realised this is a duplicate for #14010

**Comment by rijkvanzanten** [comment]
I'll mark it as a duplicate, but _yes_ I'd love to have something like this included 🙂
